src/mixins/filing-template-mixin.ts

```diff
@@ -831,9 +831,11 @@ export default class FilingTemplateMixin extends DateMixin {
     this.mutateRestorationType(filing.restoration.type)
     if (filing.restoration.expiry) {
       this.mutateRestorationExpiry(filing.restoration.expiry)
-    } else {
+    } else if (filing.restoration.type === RestorationTypes.LTD_EXTEND) {
       // Reset radio button to 2 years
       this.mutateRestorationExpiry(DateUtilities.addMonthsToDate(24, this.getStateFilingRestoration?.expiry))
+    } else {
+      this.mutateRestorationExpiry(undefined)
```

Comment:
This is not a good idea because undefined is not reactive, so something watching this (or dependent on it in a getter) won't be triggered on a future change.

Reply:
Or test it and tell me it works as-is :)

Reply:
Do we even need this line?

Reply:
At first I set it as null but when I save and then resume, I'm getting a field expiry: null. Setting it to undefined eliminates this field.

I've tested it and it works as is. This is since we only use any form of dates in extensions. Any getter that is dependent on the expiry would be there. However, I'm open to suggestions for a better way if applicable?

Reply:
`expiry` is initialized to null in the store (see state-model.ts), so if you don't set it here when parsing a draft filing then it stays at that value -- which is good.

Where do  you see "expiry: null" ? Perhaps that component needs to handle this case. Expiry isn't meaningful if the restoration type is Conversion, right?

Reply:
Yes. If restoration type is Conversion, it doesn't make sense to have expiry there. We also see that when creating a new Full Restoration in Create UI. 

This is an example of when we keep expiry as null in Conversions:
![expiry null](https://user-images.githubusercontent.com/122301442/230431167-6c02b247-6cda-4ba1-81e5-71fecff74479.PNG)

Reply:
1. What do you get if you save a draft limited resto, then you resume the draft and save a draft full resto?
2. The null expiry above may be rejected by the API. In that case you'll have to remove it (as we do with some other properties).

Reply:
1. No expiry.
![fullExpiry](https://user-images.githubusercontent.com/122301442/230435974-e39e3674-c298-4404-9758-e3ef6a166d52.PNG)

2. See next commit.